html/semantics/scripting-1/the-script-element/moving-between-documents.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Moving script elements between documents</title>
+<link rel="author" href="mailto:d@domenic.me" title="Domenic Denicola">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#execute-the-script-block">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<!-- Background:
+
+- https://www.w3.org/Bugs/Public/show_bug.cgi?id=11323
+- https://github.com/whatwg/html/issues/2137
+- https://github.com/whatwg/html/issues/2469
+- https://github.com/whatwg/html/pull/2673
+
+-->
+
+<body>
+<script>
+"use strict";
+
+for (const type of ["text/javascript", "module"]) {
+  async_test(t => {
+    t.add_cleanup(() => {
+      window.didExecute = undefined;
```

Comment:
Maybe also make this remove the iframe? (Same below.)

Reply:
Happy to do so, but why?

Reply:
I guess it's fine either way. Just seems a little cleaner to only have the results.